src/string.rs

```diff
@@ -241,7 +316,7 @@ impl<const N: usize> String<N> {
     ///
     /// unsafe {
     ///     let vec = s.as_mut_vec();
-    ///     assert_eq!(&[104, 101, 108, 108, 111][..], &vec[..]);
+    ///     assert_eq!(&b"hello", &vec);
```

Comment:
```suggestion
    ///     assert_eq!(&vec, &b"hello");
```

Reply:
Done in 6f4bc0698072e178d72cc29e82398d8a69e530b1